src/runtime_src/core/common/system.h

```diff
@@ -67,6 +67,14 @@ class system
   virtual device::id_type
   get_device_id(const std::string& str) const;
 
+  /**
+   */
+  virtual std::tuple<uint16_t, uint16_t, uint16_t, uint16_t>
+  get_bdf_info(device::id_type, bool) const
+  {
+    return std::make_tuple(uint16_t(0), uint16_t(0), uint16_t(0), uint16_t(0));
```

Comment:
```suggestion
    return {0, 0, 0, 0};
```

If that doesn't work for some reason, then use 
`std::make_tuple<uint16_t, uint16_t, uint16_t, uint16_t>(0, 0, 0, 0)`

Reply:
Unfortunately, neither of those two options will compile in the windows build.